python/tests/llm/clients/anthropic/test_anthropic_client.py

```diff
@@ -235,6 +235,199 @@ def test_call(
                     ),
                 ],
             },
+            "structured_output_basic_scenario": {
+                "provider": "anthropic",
+                "model_id": "claude-sonnet-4-0",
+                "params": None,
+                "finish_reason": FinishReason.END_TURN,
+                "messages": [
+                    SystemMessage(
+                        content=Text(
+                            text="""\
+When you are ready to respond to the user, call the __mirascope_formatted_output_tool__ tool to output a structured response.
+Do NOT output any text in addition to the tool call.\
+"""
+                        )
+                    ),
+                    UserMessage(
+                        content=[Text(text="Recommend a single fantasy book.")]
+                    ),
+                    AssistantMessage(
+                        content=[
+                            ToolCall(
+                                id="toolu_01WD5zg3AbxD1tUkgt12pP85",
+                                name="__mirascope_formatted_output_tool__",
+                                args='{"title": "The Name of the Wind", "author": "Patrick Rothfuss", "vibe": "mysterious!"}',
```

Comment:
Note: We see the `ToolCall` to `__mirascope_formatted_output_tool__` due to a bug with `BaseResponse`, which I fix upstack. The Anthropic client is functioning correctly